includes/admin/class-wc-payments-admin.php

```diff
@@ -76,6 +76,21 @@ public function add_payments_menu() {
 		);
 
 		if ( $should_render_full_menu ) {
+			if ( self::is_account_overview_page_enabled() ) {
```

Comment:
`$this->is_account_overview_page_enabled()` would it more here: the method is not static.

Reply:
Oh, I actually intended for it to be static.

Reply:
Ok, then marking as resolved - feel free to turn the method into static =)